cmd/simple/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"flag"
+	"time"
+
+	"github.com/Azure/aad-pod-identity/pkg/mic"
+	"github.com/Azure/aad-pod-identity/version"
+
+	aadpodid "github.com/Azure/aad-pod-identity/pkg/apis/aadpodidentity"
+	"github.com/Azure/aad-pod-identity/pkg/crd"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog"
+)
+
+var (
+	kubeconfig        string
+	cloudconfig       string
+	forceNamespaced   bool
+	versionInfo       bool
+	syncRetryDuration time.Duration
+	leaderElectionCfg mic.LeaderElectionConfig
+	httpProbePort     string
+)
+
+func main() {
+	defer klog.Flush()
+	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to the kube config")
+
+	flag.Set("logtostderr", "true")
+	flag.Set("v", "10")
+
+	flag.Parse()
+
+	if versionInfo {
+		version.PrintVersionAndExit()
+	}
+	klog.Infof("Starting mic process. Version: %v. Build date: %v", version.MICVersion, version.BuildDate)
```

Comment:
Since we are keeping this code in the master branch, can we update the logs to say `Starting simple process` and remove the mic related logs?

Reply:
Also, do we want to keep this in the master branch?